modules/qna/src/views/full/FormModal.jsx

```diff
@@ -178,9 +201,29 @@ export default class FormModal extends Component {
 
   deleteAnswer = index => {
     const answers = this.state.item.answers
-    if (answers[index]) {
+    const answer = answers[index]
+    if (answer) {
       answers.splice(index, 1)
       this.changeItemProperty('answers', answers)
+      //Will set as deleted if it is a contentElement and isn't used as another answer
+      if (answer.contentId && !answers.find((a, aIndex) => { console.log(a, aIndex); return a.contentId == answer.contentId })) {
+        this.setState({ deletedContentElements: [...this.state.deletedContentElements, answer.contentId] })
+      }
+    }
+  }
+
+  handleDeletedContentElements = async () => {
+    try {
+
+      if (this.state.deletedContentElements.length > 0) {
+        if (confirm("Do you want to remove the deleted content elements?")) {
+          await this.props.bp.axios.post(`/content/elements/bulk_delete`, this.state.deletedContentElements)
+        }
+      }
+
+      return true
+    } catch (e) {
+      return false;
     }
```

Comment:
Please avoid unnecessary try catch if you don't handle them or log anything.  No need to return anything if you don't use the return value also.